### DISCUSSION TITLE
Minimum Lademenge - Automatisches Laden

### DISCUSSION THREAD

**Discussion by herbert131** [goecharger/go-eCharger-API-v2#123] [Q&A]
Hallo,
ich möchte gerne über die API folgendes erreichen. Nach Eingabe der gewünschten Lademenge (in Wh) soll die WB das Fahrzeug laden und dann automatisch wieder beenden, sobald die angegebene Lademenge geladen wurde. Ich sende dazu folgende Befehle:
ate = 1000        // als Beispiel in Wh
lmo = 5           // AutomaticStop 
frc   = 2          // Ladestart

Leider beendet die WB das laden nach der angegebenen Lademenge nicht sondern läd weiter. Was hab ich vergessen?

**Comment by 0xFEEDC0DE64** [comment]
Der forceState hat höhere Priorität, bedeutet ja "Lade unter allen Umständen jetzt sofort".

kann frc weggelassen werden? Falls es um soetwas wie authorisierung geht, empfehle ich einfach auf access_control Wait zu stellen und dann eine Transkation zu starten (zb trx=0, wird keiner rfid zugeordnet, ist so als ob die app die transaktion genehmigt hätte). Achtung: trx=0 geht immer automatisch verloren beim abstecken des autos.

**Reply by herbert131** [reply]
Danke für die schnelle Antwort. Da ich zu dem besagten Zeitpunkt des aktivierens nicht weiss, ob WB läd (PV Überschuss oder nicht) kann ich dann wie folgt vorgehen:
frc = 0  // neutral oder soll ich mit frc = 1 ausschalten wenn noch geladen wird ?
trx  = 0  // hab ich bereits so implementiert
ate = 1000 // als Beispiel in Wh
lmo = 5 // AutomaticStop
.. dann sollte es funktionieren ?

**Reply by gyroslav** [reply]
mit frc=0 entscheidet die WB selber, ob sie lädt oder nicht. 
probier also 
ate = 1000
lmo = 5
frc = 0
